msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mwpj/TestCase2519783.kt

```diff
@@ -49,15 +49,17 @@ class TestCase2519783 : AbstractMsalBrokerTest() {
 
     @Test
     fun test_2519783() {
-        // Register 2 accounts from different tenants
-        mBrokerHostApp.multipleWpjApiFragment.performDeviceRegistration(mLabAccount.username, mLabAccount.password)
-        mBrokerHostApp.multipleWpjApiFragment.performDeviceRegistration(mUsGovAccount.username, mUsGovAccount.password)
-        val deviceRegistrationRecords = mBrokerHostApp.multipleWpjApiFragment.allRecords
-        Assert.assertEquals(2, deviceRegistrationRecords.size)
+        if (!mBrokerHostApp.flights.contains("\"EnableKeyStoreKeyFactory\":\"true\"")) {
```

Comment:
> if (!mBrokerHostApp.flights.contains("\"EnableKeyStoreKeyFactory\":\"true\"")) { [](http://example.com/codeflow?start=7&length=81)

You can use Assume.assumeTrue() here.

Reply:
+1

Reply:
If we "assume" this flag is not set, the test will be ignored if it is set